pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1303,6 +1303,20 @@ func parseDeviceIoListConfig(getconfigCtx *getconfigContext,
 					"key: %s, value: %s", key, value)
 			}
 		}
+
+		if ioDevicePtr.Vflist != nil && ioDevicePtr.Vflist.VfCount > 0 {
```

Comment:
@uncleDecart I'm bit confused now - I see that we have two places where config for virtual function can be specified:
- https://github.com/lf-edge/eve/blob/9.4/api/proto/config/devmodel.proto#L163
- https://github.com/lf-edge/eve/blob/9.4/api/proto/config/devcommon.proto#L52

Are they somehow merged or which one has the preference?
And how were you able to create and assign VFs if [this field](https://github.com/lf-edge/eve/blob/master/pkg/pillar/types/physicalioadapters.go#L51) is not being set by parseconfig.go (in the current EVE master)?

Reply:
fwiw, the 1st one is for the host (EVE) to define how many VFs there should be per NIC and the 2nd link is to define which VF should be passed through to the workload.